docs/tutorials/quick-start/index.rst

```diff
@@ -0,0 +1,645 @@
+an IP address from a pool. Read through the documentation from your cloud
+provider on how to arrange that.
+
+Step 3 - Assign a DNS name
+==========================
+
+The external IP that is allocated to the ingress-controller is the IP to which
+all incoming traffic should be routed. To enable this, add it to a DNS zone you
+control, for example as `example.your-domain.com`.
+
+This quickstart assumes you know how to assign a DNS entry to an IP address and
+will do so.
+
+Step 2 - Deploy an Example Service
+==================================
+
+Your service may have its own chart, or you may be deploying it directly with
+manifests. This quickstart uses manifests to create and expose a sample service.
+The example service uses `kuard`_, a demo application which makes an excellent
+back-end for examples. 
+
+The quickstart example uses three manifests for the sample. The first two are a
+sample deployment and an associated service:
+
+- deployment manifest: `deployment.yaml`_
+
+.. literalinclude:: example/deployment.yaml
+   :language: yaml
+
+- service manifest: `service.yaml`_
+
+.. literalinclude:: example/service.yaml
+   :language: yaml
+
+.. _`deployment.yaml`: https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/deployment.yaml
+.. _`service.yaml`: https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/service.yaml
+.. _`kuard`: https://github.com/kubernetes-up-and-running/kuard
+
+You can create download and reference these files locally, or you can
+reference them from the GitHub source repository for this documentation.
+To install the example service from the tutorial files straight from GitHub, 
+you may use the commands:
+
+.. code-block:: shell
+
+   $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/deployment.yaml
+   deployment.extensions "kuard" created
+
+   $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/service.yaml
+   service "kuard" created
+
+An `ingress resource`_ is what Kubernetes uses to expose this example service 
+outside the cluster.  You will need to download and modify the example manifest
+to reflect the domain that you own or  control to complete this example.
+
+
+A sample ingress you can start with is:
+
+- ingress manifest: `ingress.yaml`_
+
+.. literalinclude:: example/ingress.yaml
+   :language: yaml
+
+.. _`ingress.yaml`: https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/ingress.yaml
+.. _`ingress resource`: https://kubernetes.io/docs/concepts/services-networking/ingress/
+
+You can download the sample manifest from github with the command:
+
+.. code-block:: shell
+
+   $ curl -O https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/ingress.yaml
```

Comment:
I think newer versions of kubectl can use `kubectl create --edit` to do this. It's probably safer (from a versions point of view) to go this route though, as it'll work all the way back to Kubernetes 1.1/1.0 (whenever Ingress was added)!

Reply:
It was added in 1.1 (I recently looked it up) - will switch over to the `kubectl create --edit` setup

Reply:
Not sure if this got lost somehow too? 😄 (showing as resolved on github!)

Reply:
Missed it, getting it now